op-devstack/sysext/l1.go

```diff
@@ -26,6 +28,14 @@ func (o *Orchestrator) hydrateL1(system stack.ExtensibleSystem) {
 
 	for idx, node := range env.Env.L1.Nodes {
 		elService, ok := node.Services[ELServiceName]
+
+		// TODO: This is a hack to force direct connection to the L1 node. Tracked in #16127.
+		// Only enable this for kurtosis devnet.
+		if strings.HasPrefix(os.Getenv("DEVNET_ENV_URL"), "kt://") {
+			elService.Endpoints[RPCProtocol].ReverseProxyHeader = map[string][]string{}
+		}
+		// TODO: Remove this once we have a proper reverse proxy.
```

Comment:
```suggestion
		// TODO(#16127): This is a hack to force direct connection to the L1 node.
		// Only enable this for kurtosis devnet.
		if strings.HasPrefix(os.Getenv("DEVNET_ENV_URL"), "kt://") {
			elService.Endpoints[RPCProtocol].ReverseProxyHeader = map[string][]string{}
		}
		// TODO(#16127): Remove this once we have a proper reverse proxy.
```

We have periodic CI automation that covers this, with TODO formatted like this.